content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -223,7 +223,9 @@ data:
   lives: "3"
 ```
 
+{{< caution >}}
 When passing `--from-env-file` multiple times to create a ConfigMap from multiple data sources, only the last env-file is used:
```

Comment:
It's odd to have a colon at the end of a caution block. Can you reword to avoid that issue @christophercarney ? 

Maybe close the `{{< caution >}}` further down? I think rewording is the better approach though.